varats/varats/data/reports/blame_report.py

```diff
@@ -581,8 +577,8 @@ def translate_to_time_deltas2(
 
 
 def generate_avg_time_distribution_tuples(
-    report: tp.Union[BlameReport, BlameReportDiff], project_name: str,
-    bucket_size: int
+    report: tp.Union[BlameReport, BlameReportDiff],
+    commit_lookup: CommitLookupHelper, bucket_size: int
```

Comment:
```suggestion
    commit_lookup: CommitLookupFTy, bucket_size: int
```
Hmm, I personally prefer to end types with `Ty` or `Type` so it sticks out that this is not a normal class but a type.
So maybe: `CommitLookupFTy` or `CommitLookupTy` or `CommitLookupType`